### PR TITLE
Added multi-file picker functionality for the WLM

### LINKED ISSUE #4617: WLM multi-upload
In normal Nearby, it is natural to upload only **one** picture, because each Wikidata item should have one and only one image.

On the other hand, WLM monuments can have many pictures, the more the better.

Thus it would make sense to have Nearby's WLM pins open a multi-file picker, rather than the current single-file picker.

### PR DESCRIPTION
**Pull request by pratik2315**
**Description (required)**
It would make sense to have Nearby's WLM pins open a multi-file picker, rather than the current single-file picker.

Fixes #4617 

What changes did you make and why?
Added a condition that when the WLM checkbox is checked, the app should allow the user to select more than one picture. If the WLM is not selected it will not allow user to select more than one file.
This is needed because monuments can have more than one picture

**Tests performed (required)**
Tested beta cluster on MI Redmi 9 Prime with API level 29

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java [hunk 1]
```diff
@@ -1624,7 +1624,11 @@ private void passInfoToSheet(final Place place) {
                 Timber.d("Gallery button tapped. Place: %s", selectedPlace.toString());
                 storeSharedPrefs(selectedPlace);
 
-                controller.initiateGalleryPick(getActivity(), false);
+                if (chipWlm.isChecked()){
```
[the remark this answers is not in the document]
Comment (pratik2315) [reply app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java]: I have added that @madhurgupta10. Thanks for the info

[the remark this answers is not in the document]
Comment (pratik2315) [reply app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java]: and sorry for the mess😅

File: app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java [hunk 2]
```diff
@@ -1623,8 +1623,8 @@ private void passInfoToSheet(final Place place) {
             if (fabGallery.isShown()) {
                 Timber.d("Gallery button tapped. Place: %s", selectedPlace.toString());
                 storeSharedPrefs(selectedPlace);
-
-                controller.initiateGalleryPick(getActivity(), false);
+
```
Comment (madhurgupta10) [inline app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:1626]: Can you revert this change? :)

**Review by madhurgupta10 — CHANGES_REQUESTED** [review]

File: app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java (hunk 2 above)
Comment (pratik2315) [reply app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:1626]: So.. Should i add If-else condition like I had before? @madhurgupta10 ? 😀

Comment (madhurgupta10) [reply app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:1626]: Sorry for not being clear, can you remove the extra empty line?

**Review by madhurgupta10 — APPROVED** [review]

**Comment by madhurgupta10** [comment]
Thanks @pratik2315, congratulations on your first contribution :)

**Comment by pratik2315** [comment]
One thing i dont understand is why those build tests failed/are failing. Any idea ?

**Comment by madhurgupta10** [comment]
@pratik2315 The build seems to be failing for some other reason, I have opened a PR for it https://github.com/commons-app/apps-android-commons/pull/4646
